core/src/main/java/org/apache/any23/writer/LoggingTripleHandler.java

```diff
@@ -111,6 +116,6 @@ public void setContentLength(long contentLength) {
     }
 
     private void printHeader(PrintWriter writer) {
-        writer.println("# Document-IRI\tContent-Length\tElapsed-Time\tSuccess\tExtractors");
+        writer.println("# Document-IRI\tContent-Length\tElapsed-Time\tSuccess\tExtractors:Triples");
```

Comment:
@jgrzebyta is the intention here to print the number of triples extracted? If so then this is not current behavior with Rover. It could be default behaviour if we switched it to use counting triple handler... just a thought.